src/lib/y2storage/refinements/devicegraph_lists.rb

```diff
@@ -34,14 +34,25 @@ module DevicegraphLists
           disks:            DevicesLists::DisksList,
           partitions:       DevicesLists::PartitionsList,
           filesystems:      DevicesLists::FilesystemsList,
-          free_disk_spaces: DevicesLists::FreeDiskSpacesList
+          free_disk_spaces: DevicesLists::FreeDiskSpacesList,
+          lvm_vgs:          DevicesLists::LvmVgsList,
+          lvm_pvs:          DevicesLists::LvmPvsList,
+          lvm_lvs:          DevicesLists::LvmLvsList
+
         }
 
         DEVICE_LISTS.each do |list, klass|
           define_method(list) do
             klass.new(self)
           end
         end
+
+        alias_method :vgs, :lvm_vgs
+        alias_method :volume_groups, :lvm_vgs
+        alias_method :pvs, :lvm_pvs
+        alias_method :physical_volumes, :lvm_pvs
+        alias_method :lvs, :lvm_lvs
+        alias_method :logical_volumes, :lvm_lvs
```

Comment:
to be honest I am not such fun of having such aliases from beginning as it makes reading of code harder, because it allows writer to write any variant and for reader it is harder to remember that it is all same methods. Remember e.g. `map` and `collect` or `reduce` and `inject` or even more famous like `File.exist?` and `File.exists?`. So I suggest to pick the best name from beginning and stick to it and use alias_method only if needed for backward compatibility.